Dockerfile

```diff
@@ -16,3 +16,4 @@ RUN pipenv run python nltk_init.py
 # Install tesseract-ocr
 RUN apt-get update
 RUN apt-get install -y tesseract-ocr
+RUN apt-get -y install gcc mono-mcs
```

Comment:
What about merging it into single statement?
`RUN apt-get install -y tesseract-ocr gcc mono-mcs`